databricks-sdk-java/src/main/java/com/databricks/sdk/mixin/JobsExt.java

```diff
@@ -0,0 +1,50 @@
+package com.databricks.sdk.mixin;
+
+import com.databricks.sdk.core.ApiClient;
+import com.databricks.sdk.service.jobs.GetRunRequest;
+import com.databricks.sdk.service.jobs.JobsAPI;
+import com.databricks.sdk.service.jobs.JobsService;
+import com.databricks.sdk.service.jobs.Run;
+
+public class JobsExt extends JobsAPI {
+
+  public JobsExt(ApiClient apiClient) {
+    super(apiClient);
+  }
+
+  public JobsExt(JobsService mock) {
+    super(mock);
+  }
+
+  @Override
+  public Run getRun(GetRunRequest request) {
+    Run run = super.getRun(request);
```

Comment:
Is this calling API 2.2 or 2.1? I can't find where the version is defined

Reply:
Currently 2.1. The implementation is supposed to work with both (as long as the types are there).

https://github.com/databricks/databricks-sdk-java/blob/c7f2582f6344ada70c345e9b09d3b128d089d102/databricks-sdk-java/src/main/java/com/databricks/sdk/service/jobs/JobsImpl.java#L98

Reply:
If I understand correctly you will override the `String path` variable later?

Reply:
`JobsImpl.java` is generated. It follows that the `String path` there is also generated. My understanding is that when the version is updated on the proto, the next generation will automatically update those paths.